lib/node_modules/@stdlib/string/len/lib/constants.js

```diff
@@ -0,0 +1,68 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const CR = 0;
```

Comment:
```suggestion
var CR = 0;
```

We exclusively use `var` in order to support older JS and Node.js environments. So this needs to be changed here and elsewhere in this file.

Reply:
In fact, in this case, the individual declarations are not necessary. Can do

```javascript
var consts = {
    'CR': 0,
    'LF': 1,
    ...
};


// EXPORTS //

module.exports = consts;
```

Given that this is internal, we should not need worry about ensuring that the exported object is immutable.